app/views/common/BackboneViewHarness.vue

```diff
@@ -3,6 +3,15 @@
 </template>
 
 <script>
+  /**
+   * Note a known limitation of this harness is that it does not support swapping out the following props
```

Comment:
Could this be resolved with a watcher, multiple watchers?

Reply:
Yeah definitely I initially tried to use an all props watcher but those aren't supported.  I spent some time thinking about this and realized that for the most part I don't think that we need this functionality.  These arguments really should only be run when the component (or view) is mounted, changing them means that a significant view change is happening (swapping out the full view in the harness) or the view arguments have changed.  If the view arguments have changed they'd then be passed to the constructor which would require a full view reload.  So the semantics of the arguments changing for the View and the semantics of the props changing for the Vue component don't really line up.  If the arguments to the View constructor changes, the View would also need to support "updated" methods and there is no API for this in backbone views.   Given this I decided to add as a known limitation and wait until we need something more complex.

Tried to capture all the thoughts there but not sure I covered the nuance in how Vue components load vs how View components load, lmk if not and happy to discuss IRL.